\documentclass[basic]{inVerba-notes}
\usepackage{inVerba-math}

\newcommand{\userName}{Cullyn Newman}
\newcommand{\class}{BI:\@ 430}
\newcommand{\theTitle}{Assignment 2}
\newcommand{\institution}{Portland State University}

%chktex-file 1

  \setlist[itemize,1]{
        label=\small \( \color{minor}\bullet \),
        leftmargin=\dimexpr32pt
    }
    
\ExplSyntaxOn
\NewDocumentCommand{\sequence}{m}
 {
  \seq_set_split:Nnn \l_tmpa_seq { } { #1 }
  \seq_use:Nn \l_tmpa_seq { \linebreak[0] }
 }
\ExplSyntaxOff

\begin{document}

\begin{enumerate}\color{minor}
  \item \textbf{[ \hspace{14pt} / 5 ]} Use the DNA sequence below for the following question. Genomic DNA is double-stranded, but by convention only one of the strands is shown below. The presence of a complementary strand is assumed.
  
    \begin{quote}
      \sequence{ATGGGCAGAGACAAAAAAAATACAGCGCTTCTAGATATTGCAAGAGATATAGGAGGAGATGAAGCTGTAGAAGTTGTAAAAGCCTTAGAAAAGAAAGGAGAAGCAACAGATGAGGAATTGGCAGAATTAACTGGAGTAAGAGTTAATACGGTGAGAAAAATCTTATACGCCCTGTACGATGCTAAGCTTGCAACCTTTAGAAGAGTTAGAGATGACGAGACTGGTTGGTATTATTATTACTGGCGCATTGATACTAAAAGATTACCGGAGGTTATTAGAACAAGAAAGTTGCAAGAGCTTGAAAAGTTAAAGCAGATGCTTCAGGAAGAAACCAGCGAGACCTATTATCACTGTGGAACTCCAGGGCATCCAAAGCTAACATTTGACGAGGCCTTTGAGTACGGGTTCCAATGTCCAATATGTGGAGAGATACTTTACGAGTATGACAACTCAAAAATAATTGAAGAACTCAAAAAGCGAATTGAAGAATTAGAGATTGAACTCGGACTGAGAAGTCCACCAAAAGAAGAAAAACCAAAAAAAGCAACAAGAAGAAAAAAGTCAAGATCAGGGAAAAAGAAGAAATAA}
    \end{quote}
      
      Your lab has obtained and sequenced genomic DNA from a hyperthermophilic microbe. You want to amplify the full sequence above from the genomic DNA, using a PCR protocol.
      
      \begin{enumerate}
        \item \textbf{[ \hspace{14pt} / 2 ]} Use the Primer3Plus software (or other method of your choice) to identify two oligonucleotide primers that will amplify the entire sequence (hint: be sure to use the “Cloning” option in the upper left task selector, and be sure to select the entire sequence). Indicate the actual primer sequences, where they would anneal to the above strand (or its complement), and the 5’ and 3’ ends of each primer. Indicate the size of the DNA product expected. Are the primers ‘acceptable’ according to the software? If not, what are the potential problems with the primers?

    \item \textbf{[ \hspace{14pt} / 3 ]} You perform the PCR and run an agarose gel to view the results. You expected to see a single band for your PCR product, but you see multiple bands of different sizes, including the size that you expected. You do see bands for a marker DNA size standard, so you know that the gel and staining worked properly. What might have gone wrong with the PCR\@? Indicate changes you could make to your PCR protocol that should help to solve the problem.

  \end{enumerate}

  \item \textbf{[ \hspace{14pt} / 5 ]} Go to the \link{https://genome.ucsc.edu/cgi-bin/hgGateway}{UCSC genome browser}, select the Dec. 2013 (GRCh38/hg38) assembly, and navigate to the human CFTR gene.

  \begin{enumerate}
    \item \textbf{[ \hspace{14pt} / 1 ]} Which chromosome is the gene on, and which arm of the chromosome?
    \item \textbf{[ \hspace{14pt} / 2 ]} Use the Gencode V36 ‘full’ track to answer the following (note: this track is found in the “Genes and Gene Predictions” category and can be activated/altered if it is not initially visible). Use the gene version that is highlighted (blue background) to answer the following:
    \begin{enumerate}
      \item How many exons make up the CFTR gene?
      \item How much space does the entire gene occupy, including both exons and introns?
    \end{enumerate}

    \item \textbf{[ \hspace{14pt} / 2 ]} Activate the OMIM genes track to identify the OMIM entry for CFTR (note: this track is found in the “Phenotype and Literature” category and can be activated/altered if it is not initially visible). Visit the OMIM gene page to determine the function of CFTR, and whether it is disease associated.
    \begin{enumerate}
      \item What is the normal biological function of this gene?
      \item Which, if any, diseases is it associated with?
    \end{enumerate}
   
  \end{enumerate}

  \item \textbf{[ \hspace{14pt} / 5 ]} Prediction of protein function and conservation using BLAST\@. 
  
  Example polypeptide sequence: 
  
  \begin{quote}
    \sequence{MGRDKKNTALLDIARDIGGDEAVEVVKALEKKGEATDEELAELTGVRVNT VRKILYALYDAKLATFRRVRDDETGWYYYYWRIDTKRLPEVIRTRKLQEL EKLKQMLQEETSETYYHCGTPGHPKLTFDEAFEYGFQCPICGEILYEYDN SKIIEELKKRIEELEIELGLRSPPKEEKPKKATRRKKSRSGKKKK}
  \end{quote}

  \begin{enumerate}
    \item \textbf{[ \hspace{14pt} / 2 ]} Use \link{NCBI BLAST}{https://blast.ncbi.nlm.nih.gov/Blast.cgi} to determine the probable function of the protein encoded by the example polypeptide sequence above. Choose the Basic BLAST program “protein BLAST”. Paste the query sequence, and search the non-redundant protein sequences within “archaea” (by typing this into the “Organism” search set).
    \begin{enumerate}
      \item 
      \item 
      \item 
    \end{enumerate}
    
    \item \textbf{[ \hspace{14pt} / 1 ]} Go back to the \link{BLAST home page}{https://blast.ncbi.nlm.nih.gov/Blast.cgi}, and use standard protein BLAST determine if there are proteins with biologically relevant similarity in Homo sapiens (note: it may take a few minutes for results to arrive). 
    
    Indicate the most similar protein, and identify the organism this gene is from. To assess the significance of the alignment, give the E-value as well as the identity and positive percentages.
    
    \item \textbf{[ \hspace{14pt} / 2 ]} Next, perform a BLAST search again for similarity in Homo sapiens, but this time using the DeltaBLAST option (note: Delta stands for ‘Domain Enhanced Lookup Time Accelerated’).
    
    \begin{enumerate}
      \item Does this modify your previous answers in part b), and if so, how? How likely is it that the function of the top hit is the same as the function of the query protein sequence? Which information in the BLAST results allowed you to make this judgement?
      \item Which BLAST approach was most valuable, and why?
    \end{enumerate}
  \end{enumerate}

\end{enumerate}

\end{document}